runtime/libffi/z/ffi.c

```diff
@@ -452,20 +452,54 @@ ffi_prep_cif_machdep(ffi_cif *cif)
 
   cif->bytes = ROUND_SIZE ((n_ov * sizeof (long)) + (n_fpr * sizeof (long long)) + (n_gpr * sizeof (long)) ) + struct_size;
 /*  printf("prep_cif_machdep_cif_bytes: %d n_gpr=%d n_ov=%d n_fpr=%d\n",cif->bytes,n_gpr,n_ov,n_fpr); */
- 
+
   return FFI_OK;
 }
- 
+
+/*======================== End of Routine ============================*/
+
+/*====================================================================*/
+/*                                                                    */
+/* Name     - ffi_prep_cif_machdep_var.                               */
+/*                                                                    */
+/* Function - Perform machine dependent CIF processing for varargs.   */
+/*                                                                    */
+/*====================================================================*/
+
+ffi_status
+ffi_prep_cif_machdep_var(ffi_cif *cif, unsigned int nfixedargs, unsigned int ntotalargs)
+{
+  cif->z_nfixedargs = nfixedargs;
+  return ffi_prep_cif_machdep_core(cif);
+}
+
+/*======================== End of Routine ============================*/
+
+/*====================================================================*/
+/*                                                                    */
+/* Name     - ffi_prep_cif_machdep.                                   */
+/*                                                                    */
+/* Function - Perform machine dependent CIF processing                */
```

Comment:
Missing period.

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/pull/22396/commits/523147ec00bb79f9113161a74fb9254769aa98f5